package/gluon-l3roamd/files/etc/modules.d/gluon-l3roamd

```diff
@@ -0,0 +1 @@
+cfg80211
```

Comment:
All kernel modules will be loaded very early in the boot process, we don't need any hacks.